packages/block-editor/src/hooks/font-size.js

```diff
@@ -255,6 +255,28 @@ const withFontSizeInlineStyles = createHigherOrderComponent(
 	'withFontSizeInlineStyles'
 );
 
+const MIGRATION_PATHS = {
+	fontSize: [ [ 'fontSize' ], [ 'style', 'typography', 'fontSize' ] ],
+};
+
+export function addTransforms( result, source, index, results ) {
+	const destinationBlockType = result.name;
+	const activeSupports = {
+		fontSize: hasBlockSupport(
+			destinationBlockType,
+			FONT_SIZE_SUPPORT_KEY
+		),
+	};
+	return transformStyles(
```

Comment:
I guess we could return early if there are no active supports (same for the `color` hook).

Reply:
I added a return early check inside transformStyles so we don't need to repeat check on color and typography.